corehq/apps/receiverwrapper/tests/data/expected_form_sql.json

```diff
@@ -25,5 +25,9 @@
   "orig_id": null,
   "deprecated_form_id": null,
   "doc_type": "XFormInstance",
-  "backend_id": "sql"
+  "backend_id": "sql",
+  "commcare_version": null,
```

Comment:
Is it right that these shouldn't be included in Couch XFormInstance.to_json() @snopoke

Reply:
Or should it be removed from XFormInstanceSQL `to_json()` instead? @snopoke

Reply:
I don't think it's a big deal. Couch and SQL should be consistent though. The main impetus for adding these was to make them available in SQL. I think ideally they shouldn't be in the JSON since they are already being added by the ES pillow but if it's a big effort i'd leave it.

Side note: Probably worth changing the pillow to just use these values rather than re-calculating them (e.g. app version)

Reply:
Thanks! It was easier to remove it from SQL. Will look into the pillow

Reply:
Actually, if we don't push these meta properties out of `to_json` the ES pillow won't have access to that info, so it will have to recalculate them. Is it worth pushing these properties in the `to_json` (in both SQL and couch)